frontend/micro-ui/web/micro-ui-internals/packages/css/src/digitv2/components/selectdropdownV2.scss

```diff
@@ -4,6 +4,14 @@
   .digit-select {
     @extend .light-input-border;
     @apply relative block w-full h-10 border border-solid border-input-border;
+    &.error {
+      border: 1px solid red;
```

Comment:
add a color for error message color and user that instead of direct red

Reply:
@jagankumar-egov made the changes